test/routes/index-happy.js

```diff
@@ -242,6 +242,12 @@ lab.experiment(('All basic routes'), () => {
       strategy: 'azure-legacy',
       credentials: postLoginCredentials
     }
+  }, {
+    url: '/severity-view',
+    code: 200,
+    text: [
+      'Severity view</h2>'
```

Comment:
`</h2>` ? No opening tag.

Reply:
left the same as it matches the pattern of other unit tests above.